test/python/providers/test_fake_backends.py

```diff
@@ -104,15 +105,18 @@ def setUpClass(cls):
         optimization_level=[0, 1, 2, 3],
     )
     def test_circuit_on_fake_backend_v2(self, backend, optimization_level):
-        if not optionals.HAS_AER and backend.num_qubits > 20:
-            self.skipTest(f"Unable to run fake_backend {backend.name} without qiskit-aer")
-        job = backend.run(
-            transpile(
-                self.circuit, backend, seed_transpiler=42, optimization_level=optimization_level
-            ),
-            seed_simulator=42,
-        )
-        result = job.result()
+        with warnings.catch_warnings():
+            # TODO remove this catch once Aer stops using QobjDictField
+            warnings.filterwarnings("ignore", category=DeprecationWarning, module="qiskit")
+            if not optionals.HAS_AER and backend.num_qubits > 20:
+                self.skipTest(f"Unable to run fake_backend {backend.name} without qiskit-aer")
+            job = backend.run(
+                transpile(
+                    self.circuit, backend, seed_transpiler=42, optimization_level=optimization_level
+                ),
+                seed_simulator=42,
+            )
+            result = job.result()
```

Comment:
This catch should no longer be necessary with the base class filter, right?

Reply:
done c1c3b32caf4d040b756e3f05b08d743cefb76d5a